addons/briefing/functions/fnc_addOrbat.sqf

```diff
@@ -1,48 +1,66 @@
 /*
- * Author: PabstMirror
- * Function used to add the order of battle to player's diary
- *
- * Arguments:
- * 0: Unit to add to the OrBat to <OBJECT>
- *
- * Examples:
- * [player] call potato_briefing_fnc_addOrbat;
+ * Author: McKendrick @ SIA 
+ * Adds "ORBAT" tab to briefing that displays a list of players by their squad and roles.  Execute locally.
  *
  * Public: Yes
  */
```

Comment:
this is public and we completely break API. There are a handful of files this is not compatible with ([see](https://github.com/search?q=repo%3ABourbonWarfare%2FPOTATO%20addOrbat&type=code))